src/tests/frontend/specs/change_user_name.js

```diff
@@ -10,10 +10,12 @@ describe('change username value', function () {
     this.timeout(10000);
     await helper.toggleUserList();
     await helper.setUserName('😃');
+    // Give the server an opportunity to write the new name.
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
it would be better to spy on the websocket and continue as soon as we receive the ack to the UPDATE_USERINFO message. no big deal though, gonna merge it.

Reply:
Unfortunately we don't use socket.io acknowledgements so there's nothing to spy on. One of these days I'll get around to adding acks.